src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts

```diff
@@ -1247,19 +1264,22 @@ export class NetworkAreaDiagramViewer {
                 textNode.connectionShiftX = textNodeMoves[1].xNew;
                 textNode.connectionShiftY = textNodeMoves[1].yNew;
                 // call the node move callback, if defined
-                this.onMoveTextNodeCallback(
-                    node.equipmentId,
-                    node.svgId,
-                    textNode.svgId,
-                    textNodeMoves[0].xNew,
-                    textNodeMoves[0].yNew,
-                    textNodeMoves[0].xOrig,
-                    textNodeMoves[0].yOrig,
-                    textNodeMoves[1].xNew,
-                    textNodeMoves[1].yNew,
-                    textNodeMoves[1].xOrig,
-                    textNodeMoves[1].yOrig
-                );
+                if (callMoveTextNodeCallback) {
```

Comment:
move test over `this.onMoveTextNodeCallback` here
```
if (this.onMoveTextNodeCallback != null && callMoveTextNodeCallback) {
```

Reply:
why? still able to move node even if onMoveTextNodeCallback   (tested)